test/template_test_helpers.rb

```diff
@@ -1,12 +1,17 @@
 module TemplateTestHelpers
   def around(&block)
-    Dir.mktmpdir do |temporary_directory|
-      @partial_path = Pathname(temporary_directory).join("app", "views")
-      @config_path = Pathname(temporary_directory).join("config")
+    Dir.mktmpdir do |tmpdir|
+      temporary_directory = Pathname(tmpdir)
+
+      @partial_path = temporary_directory.join("app", "views")
+      @config_path = temporary_directory.join("config")
+      @fixture_path = temporary_directory.join("test", "clients", "fixtures")
 
       with_view_path_prefixes(@partial_path) do
         with_config_path(@config_path) do
-          block.call
+          with_fixture_path_prefixes(@fixture_path) do
```

Comment:
@georgebrock nesting three blocks deep indicates to me that I'm missing a functional concept at play here. Is it currying? Composition? How do I chain these together in Ruby?